### DISCUSSION TITLE
It is possible to set `"C_Cpp.intelliSenseCachePath"` for each system?

### DISCUSSION THREAD

**Discussion by diogo-rossi** [microsoft/vscode-cpptools#12161] [Q&A]
The default location is different for each system, as can be seen here:

![image](https://github.com/microsoft/vscode-cpptools/assets/57204023/209ef9a3-dfeb-4904-b9cf-1078e2bc2be6)

But, we can set this location with `"C_Cpp.intelliSenseCachePath"`, rigth?

![image](https://github.com/microsoft/vscode-cpptools/assets/57204023/5d55daec-84ef-4cad-a658-ef5615cc22a9)

#### My problem is:
I have to set a different location for the cache in a linux system to use a shared folder (disc space issues).
But, in another Windows machine, that location is different (it could remain default)

So, it is possible to set this path `"C_Cpp.intelliSenseCachePath"` for each system?

Or even by hostname would be good!

**Comment by sean-mcmanus** [comment]
@Diogo-Rossi You should be able to set it in the remote settings.

![image](https://github.com/microsoft/vscode-cpptools/assets/19859882/23034878-3a44-4f6e-bbef-94f97ceee04b)
